### ISSUE TITLE
August 31 Release Checklist

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC#1901]
_Hint: copy this template into a [new issue](https://github.com/18F/FEC/issues/new?title=Release:%20Sprint%20%23&body=Copy%20the%20release%20checklist%20from%20https://github.com/18F/FEC/wiki/release-checklist) to track the current release._

## Documentation
- [x] Write simplified release notes for FEC with instructions for review
- [x] Update documentation in any relevant repo

### Communciations
- [ ] Are there communications around this release from 18F or FEC?

## Reviews
- [x] 18F QA review on staging
- [x] FEC QA review on staging
- ~~File accessibility review request [here]~~ (https://github.com/18F/Accessibility_Reviews/issues/new) (still need to do, but not before release)
- [x] Verify all issues and pull requests marked "before release" are resolved

## Deployment
- [x] Flip feature flag for statutes on prod
- [x] Update fec-style and publish to npm as necessary
- [x] Run update scripts in standalone app if necessary
  - [x] Allocate additional resources for the database to help with the processing before starting if need be.
- [x] Make sure local copies of `master`, `develop`, and `release/[release name]` branches are up-to-date.
- [x] Checkout the `release/[release name]` branch.
- [x] Run `git flow release finish [release name]`
- [x] Run `git push --follow-tags`
- [x] Confirm that the new `web` app is up and running successfully.
- [x] Test the site.
- [ ] Deploy eregs

## After release
- [x] Review changes on production
- [x] Alert #fec-partners of the successful release

**Comment by LindsayYoung** [comment]
"Delete the web-a and web-b apps." is from an old release, we only needed to do that once.

Also, we will need sql updates.

**Comment by ccostino** [comment]
SQL updates have finished, we're just wrapping up our last remaining issue and then we'll get started!
